samples/tab-app-monetization/nodejs/ClientApp/appManifest/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.13",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
```

Comment:
Why we have downgraded this, can we use 1.16?

Reply:
These sample use rsc permissions, which break in latest manifest version